src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -1780,6 +1781,28 @@ public ProcessWindowStyle WindowStyle
         public SwitchParameter Wait { get; set; }
 
         /// <summary>
+        /// If specified, wait for this number of seconds
```

Comment:
Please follow common convention and add final dot in public comments.

Reply:
Added. The other public comments in `Start-Process` didn't have a dot, so I followed the same format as those.